apps/core/templates/core/content_page.html

```diff
@@ -1,11 +1,17 @@
 {% extends "base.html" %}
 
-{% load static %}
-{% load wagtailcore_tags %}
+{% load static wagtailcore_tags %}
 
```

Comment:
Why haven't we loaded our custom tag?

Reply:
Good question!

In `core_tags.py` we have:
```
from django.template.defaultfilters import register
```

This makes `@register.inclusion_tag` register as defaultfilter. But that should be:
```
from django import template

register = template.Library()
```

I'll update the pr.